package.json

```diff
@@ -42,8 +42,10 @@
     "react-dom": "15.6.1",
     "react-helmet": "5.1.3",
     "react-redux": "5.0.5",
-    "redux": "3.7.2",
+    "redux": "3.7.1",
+    "redux-form": "^7.0.0",
```

Comment:
I'm not really sure to use `redux-form`. Because I think that every new third-party library is creating more complexity to understand the boilerplate. What do you think?